packages/video_player/tizen/src/video_player.cc

```diff
@@ -422,6 +435,26 @@ void VideoPlayer::SendInitialized() {
   }
 }
 
+Eina_Bool VideoPlayer::ResetScreensaverTimeout(void *data) {
+  LOG_DEBUG("[VideoPlayer] Reset screen saver timeout.");
+
+  auto *player = static_cast<VideoPlayer *>(data);
+  if (!player->screensaver_reset_timeout_) {
+    LOG_INFO("[VideoPlayer] Symbol not found: %s", dlerror());
+    dlclose(player->screensaver_handle_);
```

Comment:
The lines 443-444 might be moved to `InitScreenSaverApi` to avoid printing the same log repeatedly. (Also `dlerror()` is not valid here.)

Reply:
Fixed.